Library/Homebrew/dev-cmd/tap-new.rb

```diff
@@ -115,6 +120,11 @@ def tap_new
             - name: Set up git
               uses: Homebrew/actions/git-user-config@master
 
+            - name: Checkout default branch
```

Comment:
This should really be handled by setup-homebrew IMO. Maybe post an issue in Homebrew/actions?

Reply:
Yes, I agree. This was sort of my workaround given that https://github.com/Homebrew/actions/pull/104 and https://github.com/Homebrew/actions/pull/109 failed. I'll open an issue.

Reply:
Yes it should do the trick

Reply:
Now that https://github.com/Homebrew/actions/pull/116 is merged (and it seems to work correctly) I've removed these lines.